pyincore/analyses/housingrecoverysequential/housingrecoverysequential.py

```diff
@@ -352,6 +416,7 @@ def housing_serial_recovery_model(
         result["guid"] = households_df["guid"]
         result["huid"] = households_df["huid"]
         result["Zone"] = households_df["Zone"]
+        result["Zone Indicator"] = households_df["Zone Indicator"]
```

Comment:
This can probably be shortened to match the 10 char limit for dbf/shapefile. Maybe zone_indic?